conjurapi/authn/oidc_authenticator.go

```diff
@@ -0,0 +1,31 @@
+package authn
+
+import (
+	"net/http"
+	"net/url"
+
+	"github.com/cyberark/conjur-api-go/conjurapi/response"
+)
+
+type OIDCAuthenticator struct {
+	AuthenticateURL string `env:"CONJUR_AUTHN_OIDC_URL"`
+	IDToken         string `env:"CONJUR_AUTHN_OIDC_ID_TOKEN"`
+}
+
+func (a *OIDCAuthenticator) RefreshToken() ([]byte, error) {
+	v := url.Values{}
+	v.Set("id_token", a.IDToken)
+
+	resp, err := http.PostForm(a.AuthenticateURL, v)
+
```

Comment:
Generally, the styleguide for this repo keeps the `err != nil` blocks tied to the same block of code so this newline can be removed.